lib/Styles/_palette.scss

```diff
@@ -72,74 +78,84 @@ $BLACK_700: #1a1a1a;
 $BLACK_900: #000;
 
 // Exported
-@define-color STRAWBERRY_100 #{$STRAWBERRY_100};
```

Comment:
We can't remove these because that would be an API break. We can only add new API